### PR TITLE
Exit

### PR DESCRIPTION
**Pull request by bun919tw**

### CODE REVIEW

File: plasma_cash/child_chain/child_chain.py [hunk 1]
```diff
@@ -68,3 +68,9 @@ def apply_transaction(self, transaction):
 
     def get_current_block(self):
         return rlp.encode(self.current_block).hex()
+
+    def get_block(self, blknum):
+        return rlp.encode(self.blocks[int(blknum)]).hex()
+
+    def get_tx_proof(self, blknum, uid):
```
Comment (boolafish) [inline plasma_cash/child_chain/child_chain.py]: I think `get_proof` would be better name?

File: plasma_cash/child_chain/child_chain.py [hunk 2]
```diff
@@ -68,3 +68,9 @@ def apply_transaction(self, transaction):
...
+        return self.blocks[int(blknum)].merkle.create_merkle_proof(int(uid))
```
Comment (boolafish) [inline plasma_cash/child_chain/child_chain.py]: I think it's better to just assume getting int as input? the int conversion is better to be done on the side whoever calls this.

**Comment by boolafish** [comment]
Can you link the issue that this solve as well

File: plasma_cash/child_chain/child_chain.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (bun919tw) [reply plasma_cash/child_chain/child_chain.py]: Yes, the complete name should be something like `get_tx_proof_in_block()` but it'll be too clumsy.

File: plasma_cash/child_chain/child_chain.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (bun919tw) [reply plasma_cash/child_chain/child_chain.py]: I'll move the int conversion to the `server.py` where the requests arrived.

File: plasma_cash/client/client.py [hunk 3]
```diff
@@ -31,3 +31,34 @@ def send_transaction(self, prev_block, uid, amount, new_owner, key):
...
+
+    def get_block(self, blknum):
+        block = self.child_chain.get_block(blknum)
+        return rlp.decode(utils.decode_hex(block), Block)
+
+    def get_tx_proof(self, blknum, uid):
+        return self.child_chain.get_tx_proof(blknum, uid)
+
+    def start_exit(self, exitor, uid, prev_tx_blk_num, tx_blk_num):
+        # TODO: Getting the whole block doesn't meet the design concept of plasma cash.
```
[the remark this answers is not in the document]
Comment (bun919tw) [reply plasma_cash/client/client.py]: Sure.

**Comment by bun919tw** [comment]
Solve issue #15 #36

**Pull request merged**